compose.yaml

```
@@ -0,0 +1,39 @@
+#     environment:
+#       - ConnectionStrings__DefaultConnection=${ConnectionStrings__DefaultConnection}
+#       - SuperAdmin__Email=${SuperAdmin__Email}
+#       - SuperAdmin__Password=${SuperAdmin__Password}
+#       - EmailSettings__Host=${EmailSettings__Host}
+#       - EmailSettings__Port=${EmailSettings__Port}
+#       - EmailSettings__Username=${EmailSettings__Username}
+#       - EmailSettings__Password=${EmailSettings__Password}
+#     volumes:
+#       - uploads-data:/app/wwwroot/uploads
+# volumes:
+#   db-data:    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)